site2/docs/io-nsq-source.md

```diff
@@ -0,0 +1,20 @@
+---
+id: io-nsq-source
+title: NSQ source connector
+sidebar_label: NSQ source connector
+---
+
+The NSQ source connector receives messages from NSQ topics 
+and writes messages to Pulsar topics.
+
+## Configuration
+
+The configuration of the NSQ source connector has the following properties.
+
+### Property
+
+| Name | Type|Required | Default | Description 
+|------|----------|----------|---------|-------------|
+| `lookupds` |String| true | " " (empty string) | A comma-separated list nsqlookupds to connect to. |
+| `topic` | String|true | " " (empty string) | The NSQ topic to transport |
+| `channel` | String |false | pulsar-transport-{$topic} | The channel to register and consume. from on the provided NSQ topic. |
```

Comment:
The sentence in the "description" column should be updated from the grammatical perspective.

Reply:
Thank you! I have altered the descriptions for `lookupds` and `channel`